File: Codeforces/1. ICPC Assiut University Community/Contest #2/A. Timon and Pumbaa/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326907/problem/A
#include <iostream>
#include <cstdio>

using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;
    cout << ((a - b) >= 0 ? (a - b) : 0) << endl;
}

File: Codeforces/1. ICPC Assiut University Community/Contest #2/A. Timon and Pumbaa/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326907/problem/A

using System;

namespace Codeforces.Assuit.Contest2
{
    class ProblemA
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' '); // take one line input
            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]); 

            Console.WriteLine(((a - b) >= 0 ? (a - b) : 0));
        }
    }
}
